pkg/custompluginmonitor/plugin/plugin.go

```diff
@@ -234,9 +234,9 @@ func (p *Plugin) Stop() {
 	glog.Info("Stop plugin execution")
 }
 
-func logPluginStderr(path, logs string) {
+func logPluginStderr(rule cpmtypes.CustomRule, logs string, logLevel glog.Level) {
 	if len(logs) != 0 {
-		glog.Infof("Start logs from plugin %q \n %s", path, string(logs))
-		glog.Infof("End logs from plugin %q", path)
+		glog.V(logLevel).Infof("Start logs from plugin %q \n %s", rule, logs)
+		glog.V(logLevel).Infof("End logs from plugin %q", rule)
```

Comment:
%q?

Reply:
%q will remove the formatting from the plugin logs and they will be less readable.